### PR TITLE
SAK-49472 T&Q ability to export the student scores

### PR DESCRIPTION
**Pull request by mpellicer**
image

Not a big fan of the button position, I contributed the patch as it was provided by UPV.

### CODE REVIEW

**Comment by mpellicer** [comment]
I'm going to check other tools and try to be consistent with the export option.

**Comment by mpellicer** [comment]
I moved the button to the top area, in the same position of the assignments tool:
image

Also moved the logic into it's own bean.

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/util/TotalScoresExportBean.java [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+	 * Creates a new TotalScoresExportBean object.
+	 */
+	public TotalScoresExportBean() {
+		log.debug("Creating a new TotalScoresExportBean");
+	}
+
+	public void exportExcel(String assessmentName, List allAgents) {
+		// Now insert the header line
+		List<String> headerList = new ArrayList<>();
+		headerList.add(ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.EvaluationMessages","first_name"));
```
Comment (bgarciaentornos) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/util/TotalScoresExportBean.java:66]: Please use a constant for the bundle string :)

Comment (mpellicer) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/util/TotalScoresExportBean.java]: Thanks see the new commit:
https://github.com/sakaiproject/sakai/pull/12108/commits/7dc85ec82c500e0c1d785c3e4ec03555d7872424

**Review by bgarciaentornos — APPROVED** [review]

**Pull request merged**